src/v2/cookbook/client-side-storage.md

```diff
@@ -101,33 +101,33 @@ const app = new Vue({
 })
 ```
 
-As before, `mounted` is used to load persisted data, if it exists. This time, though, data is only persisted when the button is clicked. We could also do any validations or transformations here before storing the value. You could also store a date representing when the values were stored. With that metadata, the `mounted` method could make a logical call on whether or not to store the values again, such as in this version below. You can try this version below.
```

Comment:
Je n'ai pas traduit la phrase suivante car ce qui est décrit n'existe pas dans l'exemple.
> With that metadata, the `mounted` method could make a logical call on whether or not to store the values again, such as in this version below.

Reply:
Est ce que cette phrase n'existe pas dans la version EN de Vue ?

Reply:
Alors il faut que j'enlève ",such as in this version below" (https://github.com/vuejs/vuejs.org/issues/1793).